megamek/src/megamek/common/MiscType.java

```diff
@@ -3644,7 +3653,7 @@ public static MiscType createSpacesuitCombatInfArmor() {
         misc.setInternalName(misc.name);
         misc.addLookupName("SpacesuitCombat");
         misc.damageDivisor = 1.0;
-        misc.subType = S_ENCUMBERING | S_SPACE_SUIT;
+        misc.subType = S_ENCUMBERING | S_SPACE_SUIT | S_XCT_VACUUM |S_COLD_WEATHER;
```

Comment:
Hate to nitpick, but |S_COLD_WEATHER should have a space between the | and the S.

Reply:
That's a tiny nit to pick lol

Reply:
"It was itchy okay?!"